sympy/physics/units/tests/test_dimensionless.py

```diff
@@ -0,0 +1,60 @@
+from sympy.physics.units import Dimension
+from sympy.physics.units import DimensionSystem
+"""
+We need to define the base quantities globally to work with and
+define the derived quantities from there on
+"""
+M = Dimension('mass')
+L = Dimension('length')
+T = Dimension('time')
+I = Dimension('current')
+dummy_system = DimensionSystem((L, M, T, I))
+def test_navier_stokes_case():
+    """ In this case my function will be able to generate one set of
+    dimensionless quantities in Navier-Stokes equations ......"""
+    Lc = L # Characteristic length scale
+    Uc = L/T # Characteristic velocity scale
+    rho_c = M/L**3 # Characteristic density scale
+    mu = M/L/T # Dynamic viscosity
+    p_c = M/L/T**2 # Characteristic pressure scale
+    F_bc = M*L/T**2 # Characteristic body forces scale
+    list_of_quantities = {'Lc': Lc, 'Uc' : Uc, 'rho' : rho_c, 'mu' : mu, 'pc' : p_c, 'Fbc' : F_bc}
+    set_of_dimless_quant, set_of_dimless_nums = dummy_system.get_dimensionless_numbers(list_of_quantities)
+    flag = True
+    for dimension in set_of_dimless_nums:
+        flag = flag and dimension == Dimension(1)
+    assert flag
+    """ You should be able to see the Reynolds number, Euler's number and Froude number in the output """
+def test_mhd_case():
```

Comment:
what about having tests with examples of electromagnetism with both SI and cgs or gaussian systems? _cgs_ and _gaussian_ dimension systems are good tests because they have one less base dimension than _SI_ (**current** is a base dimension in _SI_ but it is a derived dimension in _cgs_ and _gauss_).

Reply:
Yeah thanks for the ideas @Upabjojr Francesco! Much appreciated! Still would like to have your reviews, actually I wanted to join you guys in sympy this summer so a quicker feedback would help me work faster on the problem before the application deadline so .... And again I think the only problem lies in the .mailmap issue so if you could help ....

Reply:
No problem for GSoC. What project are you applying to?

Try not to edit the mailmap file. I don't know what the problem is.

Can you think of an application of Buckingham's Pi theorem in cgs or gaussian dimensions where the result is different than SI? This would test the stability of the code and make sure it works in dimensiom systems other than SI.

Reply:
Thanks a lot for the suggestions, would be going over them in a short while. I have a proposal by the name "Automated reduction of differential Groebner bases to simplify systems of PDEs
" which aims to implement the Kolchin-Ritt algorithm (the analogue of Buchberger's algorithm) to simplify any given system of PDEs, a topic which I have been reading extensively on for a while as a pursuit of my own research interests.

I think I remember obtaining different dimensionless quantities in Gaussian especially for the MHD case and circuit QED case, because of the vaccuum permittivity being unitless in the CGS system. But by definition, they were referring to the same quantity but with different dimensions in SI. I will still do a quick recheck just in case, not to worry! Probably if I send the print commands to the printing module, it might help. Thanks once again :)